build.sbt

```diff
@@ -39,13 +42,27 @@ lazy val core = project
     quasarPluginDatasourceFqcn := Some("quasar.datasource.kafka.KafkaDatasourceModule$"),
 
     quasarPluginDependencies ++= Seq(
-      "com.github.fd4s"         %% "fs2-kafka"            % "1.0.0",
-      "org.slf4s"               %% "slf4s-api"            % "1.7.25",
-      "org.slf4j"               %  "slf4j-simple"         % "1.7.25" % Test,
-      "org.specs2"              %% "specs2-core"          % specs2Version % Test,
-      "org.specs2"              %% "specs2-matcher-extra" % specs2Version % Test,
-      "org.specs2"              %% "specs2-scalacheck"    % specs2Version % Test,
-      "org.specs2"              %% "specs2-scalaz"        % specs2Version % Test,
-      "com.precog"              %% "quasar-foundation"    % quasarVersion.value % Test classifier "tests",
-      "io.github.embeddedkafka" %% "embedded-kafka"       % "2.5.0" % Test))
+      "co.fs2"                     %% "fs2-core"             % "2.3.0",
+      "com.github.fd4s"            %% "fs2-kafka"            % "1.0.0",
+      "com.github.julien-truffaut" %% "monocle-core"         % "1.6.0",
+      "com.precog"                 %% "quasar-api"           % quasarVersion.value,
+      "com.precog"                 %% "quasar-foundation"    % quasarVersion.value,
+      "com.precog"                 %% "quasar-frontend"      % quasarVersion.value,
+      "com.precog"                 %% "quasar-qscript"       % quasarVersion.value,
+      "com.slamdata"               %% "slamdata-predef"      % "0.1.2",
+      "io.argonaut"                %% "argonaut"             % "6.2.1",
+      "org.apache.kafka"           %  "kafka-clients"        % "2.5.0",
+      "org.scalaz"                 %% "scalaz-core"          % "7.2.30",
+      "org.slf4j"                  %  "slf4j-api"            % slf4jVersion,
+      "org.slf4s"                  %% "slf4s-api"            % slf4jVersion,
+      "org.typelevel"              %% "cats-core"            % catsVersion,
+      "org.typelevel"              %% "cats-effect"          % "2.1.2",
+      "org.typelevel"              %% "cats-kernel"          % catsVersion,
+      "com.precog"                 %% "quasar-foundation"    % quasarVersion.value % Test classifier "tests",
+      "io.github.embeddedkafka"    %% "embedded-kafka"       % "2.5.0"             % Test,
+      "org.slf4j"                  %  "slf4j-simple"         % slf4jVersion        % Test,
+      "org.specs2"                 %% "specs2-core"          % specs2Version       % Test,
+      "org.specs2"                 %% "specs2-matcher-extra" % specs2Version       % Test,
+      "org.specs2"                 %% "specs2-scalacheck"    % specs2Version       % Test,
+      "org.specs2"                 %% "specs2-scalaz"        % specs2Version       % Test))
```

Comment:
I should have caught this before, but there are two ways to declare dependencies: `quasarPluginDependencies` and `libraryDependencies`. Quoted from the README for https://github.com/precog/sbt-quasar-plugin:

> The `quasarPluginDependencies` key is analogous to `libraryDependencies`, except it will be considered as part of the assembly and packaging process for your plugin. You should declare all of your non-Test dependencies using this key rather than `libraryDependencies`

So, the test dependencies should go in `libraryDependencies` and everything else should go in `quasarPluginDependencies`.

Additionally, the plugins acquire library versions transitively via their dependency on `quasar`, and I'd prefer to use the transitive deps for a few reasons

- upgrading library deps is annoying if you have to change the version in multiple repos
- we can't actually enforce this in plugins (because you always have the transitive deps in scope)
- inevitably lib versions will get out of sync and then we have to figure out what went wrong and where

So I'd prefer if the only dependencies declared in `quasarPluginDependencies` are ones that are specific to the Kafka datasource. We haven't been entirely consistent with this everywhere, but most plugins follow this rule.